### PR TITLE
support for h-baselines

### PR DESCRIPTION
**Pull request by AboudyKreidieh**
## Pull request information

- **Status**: ? (ready to merge / in development)
- **Kind of changes**: ? (bug fix / new feature / documentation...)
- **Related PR or issue**: ? (optional)

## Description

? (general description)

### CODE REVIEW

File: docs/source/flow_setup.rst [hunk 1]
```diff
@@ -198,8 +203,11 @@ to activate the `flow` env. Type:
     source activate flow
     python examples/simulate.py ring --aimsun
 
-*Troubleshootig for Ubuntu users with Aimsun 8.4*: when you run the above example, you may get a subprocess.Popen error ``OSError: [Errno 8] Exec format error:``.  
-To fix this, go to the `Aimsun Next` main directory, open the `Aimsun_Next` binary with a text editor and add the shebang to the first line of the script ``#!/bin/sh``.
+*Troubleshootig for Ubuntu users with Aimsun 8.4*: when you run the above
```
Comment (eugenevinitsky) [inline docs/source/flow_setup.rst]: Troubleshooting

File: docs/source/flow_setup.rst [hunk 2]
```diff
@@ -267,33 +304,29 @@ You can test your installation by running
...
 
+(Optional) Install h-baselines
+------------------------------
 
-(Optional) Visualizing with Tensorboard
----------------------------------------
-
-To visualize the training progress:
+h-baselines is another variant of OpenAI's baselines that support the use of
+single-agent, multiagent, and hierarchical policies. To install h-baselines,
```
Comment (eugenevinitsky) [inline docs/source/flow_setup.rst]: this builds off stable baselines and not off baselines right? They should be credited then

File: examples/README.md [hunk 3]
```diff
@@ -10,8 +10,8 @@ within the Flow framework on a variety of traffic problems. These examples are
 python files that may be executed either from terminal or via a text editor. 
 For example, in order to execute the non-RL Ring example we run:
 
-```shell
-python simulate.py ring
+```shell script
+python simulate.py "ring"
```
Comment (eugenevinitsky) [inline examples/README.md]: does it actually need string quotes?

File: docs/source/flow_setup.rst (hunk 1 above)
Comment (AboudyKreidieh) [reply docs/source/flow_setup.rst]: I was just moving everything within 80 characters so it's easier to read

File: examples/README.md (hunk 3 above)
Comment (AboudyKreidieh) [reply examples/README.md]: it doesn't, but better practice for cases where the string might contain special characters (e.g. spaces)

File: docs/source/flow_setup.rst (hunk 2 above)
Comment (AboudyKreidieh) [reply docs/source/flow_setup.rst]: i'll switch to stable-baselines

File: examples/train.py [hunk 4]
```diff
@@ -174,89 +180,215 @@ def setup_exps_rllib(flow_params,
...
+
+    # Get the command-line arguments that are relevant here
+    args = parse_options(description="", example_usage="", args=args)
+    print(args.env_name)
+
+    # the base directory that the logged data will be stored in
+    if multiagent:
+        base_dir = "training_data/multi-fcnet"
+    else:
+        base_dir = "training_data/fcnet"
```
Comment (eugenevinitsky) [inline examples/train.py]: feel like this should go in the home directory rather than inside of flow. Amenable to different opinions

**Comment by eugenevinitsky** [comment]
Obviously should wait till tests pass.

**Comment by AboudyKreidieh** [comment]
it's not done yet. please hold off merging

**Pull request merged**